rr_iarrc/src/finish_line_watcher/finish_line_watcher.cpp

```diff
@@ -13,6 +13,7 @@
 #include <sensor_msgs/Image.h>
 #include <std_msgs/Int8.h>
 #include <climits>
+#include <cmath>
 #include <opencv2/opencv.hpp>
 
 using namespace std;
```

Comment:
Also, I know this is a bit tedious but can you remove the usage of the namespace for ros and opencv. We attempt to at most use `using namespace std;` in our codebase.